gradle.properties

```diff
@@ -0,0 +1 @@
+version=1.0.10-SNAPSHOT
```

Comment:
We already have a buildNumber in build.gradle. Eventual version released will contain that build number. How would the version as specified here work with the build number there? Asking just to make sure we're good in this aspect. Let's confirm end-to-end the process of a new release once the bintray key is defined and merge onto master starts releasing releases.

Reply:
We don't need this now. when I was working on releasing from local yesterday, I have to add this one. I will delete this file now...